src/dosbox.cpp

```diff
@@ -597,19 +597,23 @@ void DOSBOX_Init(void) {
 	Pbool = secprop->Add_bool("nosound",Property::Changeable::OnlyAtStart,false);
 	Pbool->Set_help("Enable silent mode, sound is still emulated though.");
 
-	Pint = secprop->Add_int("rate",Property::Changeable::OnlyAtStart,44100);
+	Pint = secprop->Add_int("rate", Property::Changeable::OnlyAtStart, 44100);
 	Pint->Set_values(rates);
 	Pint->Set_help("Mixer sample rate, setting any device's rate higher than this will probably lower their sound quality.");
 
-	const char *blocksizes[] = {
-		 "1024", "2048", "4096", "8192", "512", "256", 0};
-	Pint = secprop->Add_int("blocksize",Property::Changeable::OnlyAtStart,1024);
+	const char *blocksizes[] = {"1024", "2048", "4096", "8192",
+	                            "512",  "256",  "128",  0};
+	Pint = secprop->Add_int("blocksize", deprecated, 1024);
 	Pint->Set_values(blocksizes);
-	Pint->Set_help("Mixer block size, larger blocks might help sound stuttering but sound will also be more lagged.");
+	Pint->Set_help("This property is deprecated, use latency instead.");
 
-	Pint = secprop->Add_int("prebuffer",Property::Changeable::OnlyAtStart,25);
+	Pint = secprop->Add_int("prebuffer", deprecated, 25);
 	Pint->SetMinMax(0,100);
-	Pint->Set_help("How many milliseconds of data to keep on top of the blocksize.");
+	Pint->Set_help("This property is deprecated, use latency instead.");
+
+	Pint = secprop->Add_int("latency", Property::Changeable::OnlyAtStart, 20);
```

Comment:
I'm just picking this out of thin air, but do you think we could get away with 14ms as a default? 
It would let users know that we have "one frame of latency" (at DOS's maximum possible 70hz FPS). 
But if there's any risk of stuttering then absolutely don't want  to push that.. just throwing out as a suggestion for

Reply:
We really could, because 1000/70=14.29, so 15 to round it off, which would still give us:

```
MIXER: Negotiated 2-channel 44100-Hz 23ms latency audio in 1024-frame blocks
MIXER: Negotiated 2-channel 48000-Hz 21ms latency audio in 1024-frame blocks
```

Reply:
Oh; I guess latency is rounded up based on those power of two blocks?
So we'd have To default to a millisecond value that gives us a 512 block size to really stay under than 14ms frame time threshold?

(I don't want to default to 14ms if it actually comes out to 21ms.)

Reply:
Here's how the power of 2 math works out:

```
2 samples = latency: 48000kHz, 44100kHz = 0.04ms, 0.05ms
4 samples = latency: 48000kHz, 44100kHz = 0.08ms, 0.09ms
8 samples = latency: 48000kHz, 44100kHz = 0.17ms, 0.18ms
16 samples = latency: 48000kHz, 44100kHz = 0.33ms, 0.36ms
32 samples = latency: 48000kHz, 44100kHz = 0.67ms, 0.73ms
--- VIABLE ZONE ---
64 samples = latency: 48000kHz, 44100kHz = 1.33ms, 1.45ms
128 samples = latency: 48000kHz, 44100kHz = 2.67ms, 2.90ms
256 samples = latency: 48000kHz, 44100kHz = 5.33ms, 5.80ms
512 samples = latency: 48000kHz, 44100kHz = 10.67ms, 11.61ms
1024 samples = latency: 48000kHz, 44100kHz = 21.33ms, 23.22ms
2048 samples = latency: 48000kHz, 44100kHz = 42.67ms, 46.44ms
4096 samples = latency: 48000kHz, 44100kHz = 85.33ms, 92.88ms
8192 samples = latency: 48000kHz, 44100kHz = 170.67ms, 185.76ms
--- VIABLE ZONE ---
16384 samples = latency: 48000kHz, 44100kHz = 341.33ms, 371.52ms
32768 samples = latency: 48000kHz, 44100kHz = 682.67ms, 743.04ms
```

Reply:
I'm convincing myself to go with a list of "no worse than", maybe `2,3,6,12,25,50,100,200`?

Reply:
Interesting- so those select values will land them on the next power of two?

If we default to 48000 KHz rate, then I really like the idea of users actually getting the latency that they specify (if they stick with our default mixer rate).

Instead of having to do trial and error to realize that the values from 6, 7, 8, 9, 10 all give them the same latency in the log messages.

Reply:
> Interesting- so those select values will land them on the next power of two?

I'm thinking more along the lines of a naive lookup table that gives the proper power of 2.
> 
> If we default to 48000 KHz rate, then I really like the idea of users actually getting the latency that they specify (if they stick with our default mixer rate).
> 
> Instead of having to do trial and error to realize that the values from 6, 7, 8, 9, 10 all give them the same latency in the log messages.

Which, if we give them a list, we avoid, since we just can't give them exactly accurate results. Basically, we're giving them a friendlier interface to the old `blocksize`.

Reply:
Perfect 😌